src/panels/ConfigPanel/ConfigPanel.tsx

```diff
@@ -69,6 +75,23 @@ export default function ConfigPanel(): JSX.Element {
   const [savingConfig, setSavingConfig] = useState<boolean>(false);
   const [confirmType, setConfirmType] = useState<ConfirmTypes>(null);
   const [formErrors, setFormErrors] = useState<string[] | null>(null);
+  const [inlineErrors, setInlineError] = useInlineErrors({
```

Comment:
I think we should merge these two error states. To do this we would need to update `useInlineErrors` to output an error for each item if the error is an array, something like this:

```
const errors = inlineErrors.reduce<ReactNode[]>((nodes, { key, error }) => {
    const errorItems = mapping && key in mapping ? mapping[key](error) : error;
    const filteredItems = (Array.isArray(errorItems) ? errorItems : [errorItems]).filter(Boolean);
    return nodes.concat(filteredItems);
  }, []);
```

Reply:
Done!